### PR TITLE
📖 Clarify CII-Best-Practices score for each badge

### PR DESCRIPTION
**Pull request by hugovk**
#### What kind of change does this PR introduce?

(Is it a bug fix, feature, docs update, something else?)

Docs update.

- [x] PR title follows the guidelines defined in our [pull request documentation](https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process)

#### What is the current behavior?

Currently it sounds like you get the full 10 for a passing badge.

But the [code](https://github.com/ossf/scorecard/blob/29893aebc4cae462001ab19b2ec381ed670f1a92/checks/evaluation/cii_best_practices.go#L26-L62) disagrees:

* gold: 10
* silver: 7
* passing: 5
* in progress: 2

#### What is the new behavior (if this is a feature change)?**

Fix the docs to match the code.

- [n/a] Tests for the changes have been added (for bug fixes/features)

#### Does this PR introduce a user-facing change?

For user-facing changes, please add a concise, human-readable release note to
the `release-note`

(In particular, describe what changes users might need to make in their
application as a result of this pull request.)

```release-note
NONE
```

### CODE REVIEW

File: docs/checks.md [hunk 1]
```diff
@@ -161,11 +161,16 @@ which indicates that the project uses a set of security-focused best development
 source software. The check uses the URL for the Git repo and the OpenSSF Best Practices badge API.
 
 The OpenSSF Best Practices badge has 3 tiers: passing, silver, and gold. We give
-full credit to projects that meet the [passing criteria](https://bestpractices.coreinfrastructure.org/criteria/0), which is a
+full credit to projects that meet the [gold criteria](https://bestpractices.coreinfrastructure.org/criteria/1), which is a
```
Comment (hugovk) [inline docs/checks.md:164]: Oops, this should be `/2`, shall I update or would you like to (if you're doing some other magic to the commits)?

```suggestion
full credit to projects that meet the [gold criteria](https://bestpractices.coreinfrastructure.org/criteria/2), which is a
```

**Comment by spencerschrock** [comment]
Thanks for catching this. 

The contents of `docs/checks.md` are auto-generated. Can you move the changes to `docs/checks/internal/checks.yaml`?
Afterwards, run  `make generate-docs` and commit both files.

**Comment by hugovk** [comment]
Done! Spotted a typo at the same time :)

**Review by olivekl — APPROVED** [review]
LGTM, thanks for catching and fixing this!

**Pull request merged**